manifests/b/BrickLink/Studio/2.2.6_1/BrickLink.Studio.yaml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
 PackageIdentifier: BrickLink.Studio
-PackageVersion: 2.2.5_1
+PackageVersion: 2.2.6_1
```

Comment:
```suggestion
PackageVersion: 2.0
```

Reply:
This is the issue
![image](https://user-images.githubusercontent.com/15158490/124170093-4c558a00-daa7-11eb-9af9-afe4dc3a8ea3.png)
The `Publisher`, `PackageName` and `Publisher` (it looks for "Studio "wildcard"") needs to match what Windows sees, otherwise it will cause issue with `winget list` or `winget upgrade`
This is a issue that should be reported to the Bricklink team.

Reply:
I have raised this issue with the application dev team...hoping for a fix from them sooner than later. Will follow up here once it's sorted out.

Reply:
Wups didnt read this part 😁

> How did previous manifests get past this? Anyway, what is the proper way to handle this? Do we just set the version in the manifest at 2.0 and keep correcting the hash every time the software gets an update?

Previous manifests got past this because we didn't have this policy before `winget upgrade`/`winget list` got released + no quality control team

There is a feature request for product version, but its most likely gonna take a while..